zeppelin-zengine/pom.xml

```diff
@@ -625,6 +625,32 @@
         </exclusion>
       </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>io.fabric8</groupId>
```

Comment:
Would this be better ? https://github.com/kubernetes-client/java/
It is officially kubernetes java library

Reply:
hello @zjffdu - currently I'd say the fabric8 library is more mature (68 individual contributors, hundreds of releases, currently at version 3.17) and it's used by the Spark project as well. If this is an issue I can change it in one day, though - we use the fabric8 lib in-house extensively but can comment on the `kubernetes-client` one (never used). Let me know, please.

Reply:
Is `kubernetes-client` the official client? Sounds to me like we should stick to `fabric8` for the reasons @matyix gives.